app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  validates :text, presence: true
+
+  after_save :update_post_comments_counter
+
+  private
+
+  def update_post_comments_counter
+    post.update(comments_count: post.comments.count)
+  end
```

Comment:
- [OPTIONAL] Nice job implementing required changes from the previous reviewer. I see below snippet more clean. You can check it out.
```
      def update_comments_counter
          post.increment!(:comments_counter)
      end

```

You can apply the same to other models where necessary if you like it :smile: